### DISCUSSION TITLE
Kafka Consumer Offset not committed, only committed after service restarts

### DISCUSSION THREAD

**Discussion by honkuan86** [BrighterCommand/Brighter#2263] [Q&A] [answered]
Hi all,

We have noticed an issue when consuming message from Kafka using Brighter, where the offsets are not committed until a service restarts. The image below shows the logs after the service restarts, and the commitBatchSize value I've set to 10.

Is anyone having the same issue ?

![image](https://user-images.githubusercontent.com/45580206/189310972-96a6b7f9-e6c5-4b29-b22d-3853d436199b.png)

**Comment by iancooper** [comment, marked as answer]
Please use version 9.1.20, which fixes this issue
